src/System.Threading.Thread/tests/System.Threading.Thread.Tests.csproj

```diff
@@ -15,8 +15,10 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Unix_Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Unix_Release|AnyCPU'" />
   <ItemGroup>
+    <Compile Include="CompressedStackTests.cs" />
     <Compile Include="ExceptionTests.cs" />
     <Compile Include="ThreadExceptionEventArgsTests.cs" />
+    <Compile Include="ThreadTests.netstandard1.7.cs" />
```

Comment:
>ThreadTests.netstandard1.7.cs [](start = 22, length = 29)

Given that there is no file called ThreadTests.cs, then probably it doesn't make sense to add the tfm in the filename. We only do this in general when a file already exists with that name and we want to saparate netstandard1.7 tests to a different file, but that is not this case.